tests/test_s3.py

```diff
@@ -255,18 +254,18 @@ def test_list_files_order_by_last_modified(self, bucket_with_multiple_files):
     ))
     @freeze_time('2016-10-02')
     def test_save_file(
-            self,
-            empty_bucket,
-            path,
-            expected_path,
-            expected_ct,
-            expected_filename,
-            expected_ext,
-            timestamp,
-            expected_timestamp,
-            download_filename,
-            expected_cd,
-            ):
+        self,
+        empty_bucket,
+        path,
+        expected_path,
+        expected_ct,
+        expected_filename,
+        expected_ext,
+        timestamp,
+        expected_timestamp,
+        download_filename,
+        expected_cd,
+    ):
```

Comment:
Would be nice if this was just on fewer lines, if you're touching it.

Reply:
I got told off for that last time! 😂 

https://github.com/alphagov/digitalmarketplace-api/pull/601#discussion_r125472282

Thanks for the review 🙌   I'll do it 😉

Reply:
Doing it as per the top SO answer here: https://stackoverflow.com/a/24988191/3967444

Reply:
Done

Reply:
Wish I hadn't said anything 🙈 I prefer option 1 from the SO answer!

Either way though, it's better, so all good.